src/libserver/command/proto/LobbyMessageDefines.cpp

```diff
@@ -396,16 +420,57 @@ void LobbyCommandShowInventoryOK::Write(
 }
 
 void LobbyCommandShowInventoryOK::Read(
-  LobbyCommandShowInventoryOK& command,
-  SourceStream& buffer)
+  LobbyCommandShowInventoryOK& command, SourceStream& buffer)
 {
   throw std::logic_error("Not implemented.");
 }
 
+void LobbyCommandCreateNicknameNotify::Write(
+  const LobbyCommandCreateNicknameNotify& command, SinkStream& buffer)
+{
+  // Empty.
+}
+
+void LobbyCommandCreateNicknameNotify::Read(
+  LobbyCommandCreateNicknameNotify& command, SourceStream& buffer)
+{
+  throw std::runtime_error("Not implemented.");
```

Comment:
Isn't this the packet that you should read

Reply:
Notify is clientbound, OK and Cancel are serverbound. So this is correct.